components/mongodb/actions/delete-a-document/delete-a-document.mjs

```diff
@@ -0,0 +1,31 @@
+import mongodbApp from "../../mongodb.app.mjs";
+
+export default {
+  key: "mongodb-delete-a-document",
+  name: "Delete a Document",
+  description: "Delete a single document by ID. [See the docs here](https://docs.mongodb.com/manual/tutorial/remove-documents/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    mongodbApp,
+    collection: {
+      propDefinition: [
+        mongodbApp,
+        "collection",
+      ],
+    },
```

Comment:
The description for this is "Collection where the new document will be inserted". It should be something like "Collection containing the document to be deleted."